packages/expo-router/src/link/elements.tsx

```diff
@@ -0,0 +1,249 @@
+'use client';
+
+import React, { isValidElement, use, type PropsWithChildren, type ReactElement } from 'react';
+import type { SFSymbol } from 'sf-symbols-typescript';
+
+import { InternalLinkPreviewContext } from './InternalLinkPreviewContext';
+import { HrefPreview } from './preview/HrefPreview';
+import { useIsPreview } from './preview/PreviewRouteContext';
+import { NativeLinkPreviewAction, NativeLinkPreviewContent } from './preview/native';
+import { Slot } from '../ui/Slot';
+
+export interface LinkMenuActionProps {
+  /**
+   * The title of the menu item.
+   */
+  title: string;
+  /**
+   * Optional SF Symbol displayed alongside the menu item.
+   */
+  icon?: SFSymbol;
+  /**
+   * If true, the menu item will be disabled and not selectable.
+   *
+   * https://developer.apple.com/documentation/uikit/uimenuelement/attributes/disabled
```

Comment:
```suggestion
   * @see [Apple documentation](https://developer.apple.com/documentation/uikit/uimenuelement/attributes/disabled) for more information.
```

These links don't give much context since they are isolated. I'd suggest providing context to the links above.